stixpy/net/client.py

```diff
@@ -35,16 +38,31 @@ class STIXClient(GenericClient):
     <BLANKLINE>
     """
 
-    baseurl = r"https://pub099.cs.technik.fhnw.ch/data/fits/" r"{level}/{year:4d}/{month:02d}/{day:02d}/{datatype}/"
-    ql_filename = r"solo_{level}_stix-{product}_\d{{8}}_V\d{{2}}\D?.fits"
-    sci_filename = r"solo_{level}_stix-{product}_" r"\d{{8}}T\d{{6}}-\d{{8}}T\d{{6}}_V\d{{2}}\D?_.*.fits"
+    baseurl = r"https://pub099.cs.technik.fhnw.ch/data/fits/"
+    datapath = r"{level}/{year:4d}/{month:02d}/{day:02d}/{datatype}/"
+
+    ql_filename = r"solo_{level}_stix-{product}_[0-9]{{8}}_V[0-9]{{2}}[a-zA-Z]?.fits"
+    sci_filename = (
+        r"solo_{level}_stix-{product}_" r"[0-9]{{8}}T[0-9]{{6}}-[0-9]{{8}}T[0-9]{{6}}_V[0-9]{{2}}[a-zA-Z]?_.*.fits"
+    )
```

Comment:
- Did the changes to `ql_filename` an `sci_filename` actually affect matching?  Your commit message implies that you made these changes for Windows, but I don't immediately see how that would be the case.
- The period in `.fits` should formally be escaped